### PR TITLE
Editorial improvements

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

- uses field instead of header
- fix normative references

### CODE REVIEW

File: draft-ietf-httpapi-ratelimit-headers.md [hunk 1]
```diff
@@ -62,9 +59,10 @@ The widespreading of HTTP as a distributed computation protocol
 requires an explicit way of communicating service status and
 usage quotas.
 
-This was partially addressed with the `Retry-After` header field
+This was partially addressed by the `Retry-After` header field
 defined in {{!SEMANTICS=I-D.ietf-httpbis-semantics}} to be returned in
-`429 Too Many Requests` or `503 Service Unavailable` responses.
+`429 Too Many Requests` (see {{?STATUS429=RFC6525}})
+or `503 Service Unavailable` responses.
```
Comment (unleashed) [inline draft-ietf-httpapi-ratelimit-headers.md:65]: No link for 503?

**Review by unleashed — APPROVED** [review]
LGTM

File: draft-ietf-httpapi-ratelimit-headers.md (hunk 1 above)
Comment (ioggstream) [reply draft-ietf-httpapi-ratelimit-headers.md:65]: 503 is in semantics... I think it's redundant but if you think it helps I can add it.

Comment (unleashed) [reply draft-ietf-httpapi-ratelimit-headers.md:65]: Just mentioned it for the sake of consistency with 429, but indeed not needed.

**Pull request merged**